convert2rhel/pkghandler.py

```diff
@@ -173,8 +173,8 @@ def call_yum_cmd(
     return stdout, returncode
 
 
-def get_problematic_pkgs(output, excluded_pkgs=frozenset()):
-    """Parse the YUM/DNF output to find what packages are causing a transaction failure."""
+def get_problematic_pkgs(output, excluded_pkgs=set()):
```

Comment:
@bocekm, When I rebased this and resolved the conflicts, this line changed from `frozenset()`to `set()` this is right? I didn't saw this PR earlier, so I don't remember if this is intended.

Reply:
Ok! gonna revert to the original. Thanks